pkg/server/status.go

```diff
@@ -40,10 +41,18 @@ func (c *criContainerdService) Status(ctx context.Context, r *runtime.StatusRequ
 		Type:   runtime.NetworkReady,
 		Status: true,
 	}
-	if err := c.netPlugin.Status(); err != nil {
-		networkCondition.Status = false
-		networkCondition.Reason = networkNotReadyReason
-		networkCondition.Message = fmt.Sprintf("Network plugin returns error: %v", err)
+	if c.netPlugin == nil {
```

Comment:
Why `netPlugin` can be nil here?

There is a race condition of `netPlugin` here? I prefer a `Status` or `Sync` call from go-cni instead of always `New`.